script/create_installer_archive_helper.py

```diff
@@ -7,6 +7,7 @@
 
 import os
 import shutil
+from sign_binaries import sign_binaries, sign_binary
```

Comment:
Wasn't sure if this was intentional?

Reply:
Pylint complained about the line, because I edited this file.